components/Image/Image.tsx

```diff
@@ -1,7 +1,7 @@
 // Code taken from GitHub discussion thread on Next/Image
 // https://github.com/vercel/next.js/discussions/18739
 
-import NextImage, { ImageProps } from 'next/image';
+import NextImage, { ImageProps } from 'next/legacy/image';
```

Comment:
I'm guessing this is another legacy change? I couldn't actually find NextImage in the Next docs

Reply:
The legacy image is hidden away - to find out more about NextImage, the component is typically referred to just as `Image`, and NextImage is just a different way of importing it and using it in your code. Here's a link to the legacy image documentation: https://nextjs.org/docs/pages/api-reference/components/image-legacy